src/client/Footer.jsx

```diff
@@ -0,0 +1,39 @@
+import { Link } from "react-router-dom"
+import {FaFacebook, FaInstagram} from 'react-icons/fa';
+
+const Footer = () => {
+  return (
+		<footer className="footer-container">
+			<div className="footer">
+        <Link to="/" className="footer__logo">Pokimo & King</Link>
```

Comment:
The tabbing makes it a bit hard to read/understand what's going on inside the div. Not sure if it it GitHub's formatting or if it's like this in the actual file

Reply:
Thanks for the feedback. I fixed it.

Reply:
I agree with Alexa ;) Glad you fixed it, @hannah68 :)